core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -1777,7 +1799,7 @@ private[spark] class BlockManager(
 
   def decommissionBlockManager(): Unit = {
     if (!blockManagerDecommissioning) {
-      logInfo("Starting block manager decommissioning process")
+      logInfo("Starting block manager decommissioning process...")
       blockManagerDecommissioning = true
       decommissionManager = Some(new BlockManagerDecommissionManager(conf))
       decommissionManager.foreach(_.start())
```

Comment:
I don't find `decommissionBlockManager` in current master branch?

Reply:
Yeah this was added in the cache block migration PR (now merged)